src/components/common/NetworkSelector/index.tsx

```diff
@@ -179,46 +181,6 @@ const UndeployedNetworks = ({
   )
 }
 
-const DeployedNetworkMenuItem = ({
-  chainId,
-  chainConfigs,
-  isSelected = false,
-  onClick,
-  safeOverviews,
-  getNetworkLink,
-}: {
```

Comment:
@clovisdasilvaneto I had to move this component back to be a function call because the network selector wasn't closing anymore. From what I could find it is because MUI Select expects the first child to be a `MenuItem` so extracting it into a custom component broke that. I tried adding a `forwardRef` without success. Do you know of anything we could still try?